kibitzr/notifier/ntfy.py

```diff
@@ -0,0 +1,48 @@
+import logging
+
+from ..conf import settings
+
+
+logger = logging.getLogger(__name__)
+
+
+class NtfyNotify:
+    def __init__(self, value):
+        import requests  # pylint: disable=import-outside-toplevel
+        self.session = requests.Session()
+        ntfy_creds = settings().creds['ntfy']
+        self.url = ntfy_creds.get('url')
+        self.topic = ntfy_creds.get('topic')
+        self.auth = ntfy_creds.get('auth', None)
+        if self.auth:
+            self.session.headers.update({'Authorization': self.auth})
+        self.topic = ntfy_creds.get('topic')
+        if value is None:
+            value = {}
+        if 'topic' in value:
+            self.topic = value.get('topic')
+        self.title = value.get('title', 'kibitzr')
+        self.priority = value.get('priority', 3)
+
+    def post(self, report):
+        logger.info("Executing ntfy notifier")
+        response = self.session.post(
+            url=self.url,
+            json=self.json(report),
+        )
+        logger.debug(response.text)
+        response.raise_for_status()
+        return response
```

Comment:
Should it return response? I thought notifiers' results are ignored.

Reply:
I copied the code from Gotify, which appears to have copied from Webhook. I think you're right, though, that the results are ignored. I'll remove it from here. Want me to remove it from the others as well?